src/fsharp/FSharp.Core.Unittests/FSharp.Core/Microsoft.FSharp.Core/StringInterpolationTests.fs

```diff
@@ -0,0 +1,35 @@
+// Copyright (c) Microsoft Corporation.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+namespace FSharp.Core.Unittests.FSharp_Core.Microsoft_FSharp_Core
+
+open System
+open FSharp.Core.Unittests.LibraryTestFx
+open NUnit.Framework
+
+type A(s: string) = 
+    member val Y = s
+
+type B = {
+    X : string
+    Y : int
+}
+
+
+[<TestFixture>]
+type StringInterpolationTests() =
```

Comment:
I think these test should be in Compiler unit tests.
FSharp.Core shouldn't know about string interpolation.